var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -33,6 +33,7 @@ class Cp2k(MakefilePackage, CudaPackage, CMakePackage, ROCmPackage):
 
     maintainers("dev-zero", "mtaillefumier")
 
+    version("2023.2", sha256="90b6d777aeac5e18f15dad4a61181c575ca023783d484740e2728a0e05169188")
```

Comment:
I am not getting this sha256 from `spack checksum`.  How did you calculate it?

Reply:
See https://spack.readthedocs.io/en/latest/packaging_guide.html#spack-checksum.

Reply:
I got the checksum after downloading the official tar release of cp2k and used sha256sum.

Reply:
This command line is practical. It saves me a lot of time. Thanks.
The next commit should fix the issue